provisioning/miniprov/hare_mp/main.py

```diff
@@ -225,6 +227,42 @@ def init(args):
         raise RuntimeError(f'Error while initializing cluster :key={error}')
 
 
+def start_mkfs(hostname: str, hare_config_dir: str):
+    motr_config_dir = '/etc/motr'
+    os.makedirs(motr_config_dir, exist_ok=True)
+    shutil.copy(f'{hare_config_dir}/confd.xc', motr_config_dir)
+    src = f'{hare_config_dir}/sysconfig/motr/{hostname}'
+    for file in os.listdir(src):
+        shutil.copy(os.path.join(src, file),'/etc/sysconfig')
+    MOTR_NODE_UUID = str(uuid.uuid4())
+    contents = f"MOTR_NODE_UUID='{MOTR_NODE_UUID}'"
+    save('/etc/sysconfig/motr-kernel', contents)
+
+    generator = Generator(hostname, hare_config_dir,
+                          kv_file=f'{hare_config_dir}/consul-kv.json')
+    cmd = '/usr/libexec/cortx-motr/motr-mkfs'
+    # start mkfs for confd, ios services
+    for svc in ('confd', 'ios'):
+        svc_fids = generator.get_svc_fids(svc)
+        for fid in svc_fids:
+            execute([cmd, fid])
+
+
+def init(args):
+    try:
+        rc = 0
+        url = args.config[0]
+        # TODO
+        # start_hax()
```

Comment:
https://github.com/Seagate/cortx-hare/pull/1801 is merged, so now you can start hax and consul here. See https://github.com/Seagate/cortx-hare/pull/1808.

Reply:
done